vhd/packer/install-dependencies.sh

```diff
@@ -68,7 +68,7 @@ echo "  - apmz $apmz_version" >> ${VHD_LOGS_FILEPATH}
 installBpftrace
 echo "  - bpftrace" >> ${VHD_LOGS_FILEPATH}
 
-MOBY_VERSION="3.0.13"
+MOBY_VERSION="19.03.12"
 installMoby
```

Comment:
@cpuguy83 add the working "systemctl start docker" command here below `installMoby`

Reply:
Adding this here so I can directly commit. :)

```suggestion
installMoby
systemctl start docker
```